conf/bkenv.sh

```diff
@@ -32,15 +32,21 @@ BOOKIE_CONF=${BOOKIE_CONF:-"$BK_HOME/conf/bookkeeper.conf"}
 # BOOKIE_LOG_CONF=
 
 # Logs location
-# BOOKIE_LOG_DIR=
+# PULSAR_LOG_DIR=
```

Comment:
Is it a related change?

Others look good to me.

Reply:
not related @codelipenghui。 Should I open an new pr?

Reply:
yes, would be better to revert the change since it doesn't look related to JAVA_HOME changes